ManaTech-Group-4/KaLunWA
# Test Cases for Posting an Image Record

Tests:
- Expected fields 
  - check status 201 (created)
  - check data returned after creation
- No image attached on form-data
  - 404 error, Code: required
- Invalid image uploaded (not in bytes e.g. string)
  - 404 error, Code: invalid
